docs/src/verification-results.md

```diff
@@ -94,3 +94,66 @@ Check 2: undetermined_example.assertion.2
          - Status: UNDETERMINED
          - Description: "assertion failed: x == 0"
 ```
+
+## Cover property results
+
+Kani provides a [`kani::cover`](https://model-checking.github.io/kani/crates/doc/kani/macro.cover.html) macro that can be used for checking whether a condition may occur at a certain point in the code.
+
+The result of a cover property can be one of the following:
+
+1. `SATISFIED`: This indicates that Kani found an execution that triggers the specified condition.
+
+The following example uses `kani::cover` to check if it's possible for `x` and `y` to hold the values 24 and 72, respectively, after 3 iterations of the `while` loop, which turns out to be the case.
+```rust
+{{#include getting-started/verification-results/src/main.rs:cover_satisfied_example}}
+```
+Results:
+```
+Check 1: cover_satisfied_example.cover.1
+         - Status: SATISFIED
+         - Description: "cover condition: i > 2 && x == 24 && y == 72"
+         - Location: src/main.rs:60:9 in function cover_satisfied_example
+```
+
+2. `UNSATISFIABLE`: This indicates that Kani _proved_ that the specified condition is impossible.
+
+The following example uses `kani::cover` to check if it's possible to have a UTF-8 encoded string consisting of 5 bytes that correspond to a string with a single character.
+```rust
+{{#include getting-started/verification-results/src/main.rs:cover_unsatisfiable_example}}
+```
+which is not possible as such string will contain at least two characters.
+```
+Check 46: cover_unsatisfiable_example.cover.1
+         - Status: UNSATISFIABLE
+         - Description: "cover condition: s.chars().count() <= 1"
+         - Location: src/main.rs:75:9 in function cover_unsatisfiable_example
+```
+
+3. `UNREACHABLE`: This indicates that the `cover` property itself is unreachable (i.e. it is _vacuously_ unsatisfiable).
```

Comment:
I would add something here like:

> In contrast to an `UNREACHABLE` result for assertions, an unreachable cover check is considered a proof failure.

Reply:
Thanks. I weakened the statement a bit, since we don't fail verification if a cover property is unreachable or unsatisfiable:

> In contrast to an `UNREACHABLE` result for assertions, an unreachable (or an unsatisfiable) cover property may indicate an incomplete proof.

Good?

Reply:
Oh, I thought that was largely the point of cover, but if that's not the behavior then that looks good for the doc.

Reply:
Yeah, that was originally the plan, but the discussion [here ](https://github.com/model-checking/kani/pull/1906#issuecomment-1318036336)led to changing it.